docs/dev-guide/getting-started.md

```diff
@@ -30,7 +30,46 @@ Download [Eclipse IDE for Enterprise Java Developers](https://www.eclipse.org/do
 
 Git fork and clone the quality measure and cohort engine repository using GitHub Standard Fork & Pull Request Workflow. For instance, ``git clone git@github.com:<myuser>/Alvearie/quality-measure-and-cohort-service.git`` where ``myuser`` is your GitHub user or GitHub organization.
 
-Once cloned locally, execute the following from the root of the repository:
+The code in the quality-measure-and-cohort-service project has dependencies on code in the https://github.com/Alvearie/rest-service-framework project. In order to build, we must set up access to the snapshot packages in the rest-service-framework project. To do this, you must first create a Personal Access Token by going to https://github.com/settings/tokens and logging in with your github userid. Click "Personal access tokens" in the left side menu, then click the "Generate new token" button. Under the "Select Scopes," check read:packages, write:packages and delete:packages and click the "Generate Token" button. Copy the generated token string as it will be used in the settings.xml updates below.
```

Comment:
Should we be recommending write and delete packages for personal tokens?

Reply:
Good point, confirmed it works with just read:packages and will remove the others